flake.nix

```diff
@@ -34,8 +34,15 @@
           let
             pname = "statix";
             packageMeta = (lib.importTOML ./bin/Cargo.toml).package;
+            toolchain = (rustChannel final).withComponents [
+                "cargo"
+                "rustc"
+                "rust-std"
+                "clippy"
+            ];
             rustPlatform = makeRustPlatform {
-              inherit (rustChannel final) cargo rustc;
+                cargo = toolchain;
+                rustc = toolchain;
```

Comment:
Sorry I'm not familiar. Is this an adoption of a more modern API?

Reply:
yeah. there are multiple ways to call `makeRustPlatform`. Reverted back to the previous inherit way and just added clippy.